### ISSUE TITLE
I get error when login In first time access after installation

### ISSUE THREAD

**Issue by vieckzand** [jesusantguerrero/sunday#93]
I get error like this after i press login button.
![image](https://user-images.githubusercontent.com/1054904/164593550-4fbc0ed7-c11d-46de-b221-4d55bb390919.png)

**Comment by jesusantguerrero** [comment]
Thanks for reporting the issue?

Did you install it manually or via docker? if it was manually can you tell me your DB version?
I know what line make that SQL call but I couldn't reproduce the error

**Comment by vieckzand** [comment]
i'm install manually. I use centos 8 Stream, php 8.1.5
![image](https://user-images.githubusercontent.com/1054904/164889229-e196473a-c987-449b-82a9-d54f0fcdef27.png)

![image](https://user-images.githubusercontent.com/1054904/164889087-a79ba322-cb08-40d3-bf7a-1ea72209484c.png)

when i execute nom install, i got error like this picture
![image](https://user-images.githubusercontent.com/1054904/164892068-c6a2ce99-eef3-4683-b66d-4be0e4e27481.png)

I interest to try your application, can you guide me how to install in detail? Thank you

**Comment by jesusantguerrero** [comment]
Thank you for the interest, I appreciate it.

The readme is up to date with the installation steps if your getting the sql error, maybe is because I tested in a different version of MySql.

I updated the repo yesterday to fix the query error. And about the `npm install` vulnerabilities you dont need it on the server it is already compiled you would only for development changes.

The steps are

Setup Without Docker
```bash
# composer install
composer install --ignore-platform-reqs
# generate key
php artisan key:generate
# run migrations
php artisan migrate
# run seeds
php artisan db:seed
```
The go to login again and create an account. Let me know if you find any issue to see if I can make a video or something. Thanks

**Comment by vieckzand** [comment]
Thanks for your response. Now I can run and login perfectly. But some features not run perfectly like Team Settings and Notes
![image](https://user-images.githubusercontent.com/1054904/164965811-7776aaee-e53c-4a34-9ee7-9c50317f0898.png)
![image](https://user-images.githubusercontent.com/1054904/164965820-322cc24a-69d7-4b45-9f2d-283cb1ed5af1.png)

**Comment by jesusantguerrero** [comment]
Thanks for pointing this out.

I started the project almost the same week after jetstream 2 years ago. It seems like invitations were added after because in my local I don't have `team_invitations` migration.

Sunday is unfinished at the time and only the features in the readme are completed. (tasks, pomodoro, timer, boards and stages) Gmail and Calendar integation are partially completed too. but with things like note, habits, Eisenhower Matrix I didn't finish the idea because I started a more personal/individual project [zen](https://zenboard.app/).

Anyways It seems like many people are interested in the concept I will consider going back and finish those ideas.
